app/views/exams/StartPage.vue

```diff
@@ -83,9 +85,9 @@ export default {
     ...mapActions('exams', [
       'startExam',
     ]),
-    async startExam () {
+    async localStartExam () {
       if (this.newUser) {
-        await this.startExam(this.examId)
+        await this.startExam(this.examId, { codeLanguage: this.codeLanguage })
```

Comment:
As soon as I clicked on start exam, it ended for me and took me to last screen. In the backend, endDate was set close to startDate for some reason too.
<img width="353" alt="Screenshot 2024-10-21 at 12 59 46 PM" src="https://github.com/user-attachments/assets/86e9ddbf-35f3-4333-aaad-ff427d564d7f">

Reply:
I dont see the codeLanguage being passed in the API request even